lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -709,6 +709,10 @@ LogicalResult NEQPrimOp::canonicalize(NEQPrimOp op, PatternRewriter &rewriter) {
 //===----------------------------------------------------------------------===//
 
 OpFoldResult AsSIntPrimOp::fold(ArrayRef<Attribute> operands) {
+  // No effect.
+  if (input().getType() == getType())
+      return input();
```

Comment:
Is this properly clang formatted?  It looks like indentation by 4 before the return?

Reply:
I'll look in to it

Reply:
After re-running clang format locally, it fixed the indentation error.  I have no idea why the pre-submit CI test didn't catch it though.  I pushed the fix directly to main.

Reply:
Thanks, I'm also curious why CI didn't see it.

Reply:
Spotted an issue in finding the base commit and opened an issue https://github.com/llvm/circt/issues/1379